Src/Shared/Core/Logger.cpp

```diff
@@ -33,16 +33,14 @@ CLog::CLog()
 CLog::~CLog()
 {
 #ifndef __WIN32
-
 	// Reset
 	Log(EType::None, CONSOLE_RESET);
-
 #endif
 }
 
 void CLog::Log(EType Flags, std::string Text)
 {
-	m_pActive->Send([this, Flags, &Text]() { LogText(Flags, std::move(Text)); });
+	m_pActive->Send([this, Flags, Text]() { LogText(Flags, std::move(Text)); });
```

Comment:
you probably want `Text = std::move(Text)` but IIRC that's C++14...

Reply:
Yup, there's an extra copy here, but performance really doesn't matter enough to use move semantics in the first place. Really shouldn't have micro-optimized like this when I wrote the underlying logger code almost a decade past. :/

For now just focusing on getting the thing to be usable.